docs/api/tab.md

```diff
@@ -22,7 +22,7 @@ import EncapsulationPill from '@components/page/api/EncapsulationPill';
 The tab component is a child component of [tabs](tabs.md). Each tab can contain a top level navigation stack for an app or a single view. An app can have many tabs, all with their own independent navigation.
 
 :::note
- Note: This component should only be used with vanilla JavaScript projects. For Angular, React, and Vue apps you do not need to use `ion-tab` to declare your tab components.
+This component can only be used when the `ion-tabs` component is set for [basic usage](./tabs.md#basic-usage). When setting up tabs with routing, the `ion-tab` component cannot be used.
```

Comment:
I don't understand this note. The vanilla JavaScript example still uses an `ion-tab` component with routing: https://ionic-docs-git-rou-11138-ionic1.vercel.app/docs/api/tabs#usage-with-router

Reply:
https://github.com/ionic-team/ionic-docs/pull/3824/commits/3fe3f9390dee9b3bff2058342c07556bc8ff1b17